geonode/urls.py

```diff
@@ -94,6 +94,7 @@
     re_path(r"^account/", include("allauth.urls")),
     re_path(r"^invitations/", include("geonode.invitations.urls", namespace="geonode.invitations")),
     re_path(r"^people/", include("geonode.people.urls")),
+    re_path(r"^people/", include("geonode.people.api.urls")),
```

Comment:
@RegisSinjari Why did you move the api/v2 urls under `/people`, instead of registering them in the api/v2 router as it was [before](https://github.com/GeoNode/geonode/pull/12081/files#diff-6010df00da7acacc61cb34695618f9993793f8077265dccbe839fb1cc58cab42L29)?